### PR TITLE
tracing/zipkin updates (new kafka collector)

### PR DESCRIPTION
**Pull request by basvanbeek**
This contains pull request #161 with updates as requested in the code comments. I have pulled in the request as the original author has not responded for almost a month and besides minor updates it looks good to go.

It also includes a bugfix to the Zipkin Endpoint.ipv4 value and has an updated AnnotateBinary() function allowing more go types to be used as annotation values serializing to the various available Zipkin binary annotation types.

Next up is an updated README to explain how one can set-up Zipkin with Go kit and Kafka. Since Scribe as collector transport for Zipkin is deprecated, I'd say one specifically needing Scribe will know how to get that up and running :)

### CODE REVIEW

**Comment by peterbourgon** [comment]
Brilliant, thank you! Looking forward to the final version :)

File: tracing/README.md [hunk 1]
```diff
@@ -15,6 +15,66 @@ hot spots, and diagnosing errors. All microservice infrastructures will
...
+
+First make sure you've installed [Vagrant] on your machine. Then you can
+download and run the [Vagrant] image like this from the command line:
+
+```
+# create a new directory to store your vagrant configuration and image files
+mkdir zipkin
+cd zipkin
+vagrant init bas303/zipkin
+vagrant-up --provider virtualbox
```
Comment (peterbourgon) [inline tracing/README.md]: `vagrant up` I think?

**Comment by basvanbeek** [comment]
yes without the dash... typo which I'll fix

**Comment by peterbourgon** [comment]
Thank you! I've filed #224 as a followup.

**Pull request merged**